llvm/lib/Target/WebAssembly/WebAssemblyInstrSIMD.td

```diff
@@ -1144,6 +1144,9 @@ def : Pat<(wasm_shr_u
               (vec.splat (i32 1))),
             (i32 1)),
           (inst $lhs, $rhs)>;
+
+def : Pat<(vec.vt(avgceilu(vec.vt V128:$lhs), (vec.vt V128:$rhs))), (inst $lhs,
+                                                                        $rhs)>;
```

Comment:
Nit, the spacing is a bit off here, probably best to split this into one line for the input and one for the output

```suggestion
def : Pat<(vec.vt (avgceilu (vec.vt V128:$lhs), (vec.vt V128:$rhs))),
          (inst $lhs, $rhs)>;
```